bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeSystem.scala

```diff
@@ -3,25 +3,91 @@ package edu.gemini.spModel.core
 import scala.collection.JavaConversions._
 import scalaz.Equal
 
-sealed abstract class MagnitudeSystem(val name: String) extends Product with Serializable
+/** Unit that can represent an integrated brightness or a uniform surface brightness per area. */
+sealed trait BrightnessUnit extends Product with Serializable {
+  val name: String
+  def displayValue: String = name
+}
 
+sealed abstract class MagnitudeSystem(val name: String) extends BrightnessUnit
+
+sealed abstract class SurfaceBrightness(val name: String) extends BrightnessUnit
+
+/** Units for integrated brightness values.
+  * Note that strictly speaking these are not all magnitudes, only Vega Mag and AB Mag are magnitudes, but
+  * for historical reasons all of these units are called magnitudes. A better term for these units would be
+  * integrated brightness units.
+  */
 object MagnitudeSystem {
 
-  case object Vega   extends MagnitudeSystem("Vega")
-  case object AB     extends MagnitudeSystem("AB")
-  case object Jy     extends MagnitudeSystem("Jy")
+  case object Vega                extends MagnitudeSystem("Vega")
+  case object AB                  extends MagnitudeSystem("AB")
+  case object Jy                  extends MagnitudeSystem("Jy")
+
+  // currently not supported in OT, only available in ITC web app
+  case object Watts               extends MagnitudeSystem("W/m²/µm")
+  case object ErgsWavelength      extends MagnitudeSystem("erg/s/cm²/Å")
+  case object ErgsFrequency       extends MagnitudeSystem("erg/s/cm²/Hz")
+
 
   val default: MagnitudeSystem = Vega
 
   val all: List[MagnitudeSystem] =
+    List(Vega, AB, Jy, Watts, ErgsWavelength, ErgsFrequency)
+
+  // this is used in the OT, for now we only support Vega, AB and Jy in the OT
+  val allForOT: List[MagnitudeSystem] =
     List(Vega, AB, Jy)
 
+  def fromString(s: String): Option[MagnitudeSystem] = s match {
+    // these strings are currently used to represent MagnitudeSystem values in the ITC web forms
+    // in case they need to be changed, please also update the science regression test scripts
+    case "MAG"                  => Some(MagnitudeSystem.Vega)
+    case "ABMAG"                => Some(MagnitudeSystem.AB)
+    case "JY"                   => Some(MagnitudeSystem.Jy)
+    case "WATTS"                => Some(MagnitudeSystem.Watts)
+    case "ERGS_WAVELENGTH"      => Some(MagnitudeSystem.ErgsWavelength)
+    case "ERGS_FREQUENCY"       => Some(MagnitudeSystem.ErgsFrequency)
+    case _                      => None
+  }
+
+  def unsafeFromString(s: String): MagnitudeSystem = fromString(s).get
```

Comment:
I would expect `fromString(sys.name) == Some(sys)` to hold but this doesn't seem to be the case (`MAG` vs `Vega` for instance). If this is a non-canonical mapping then it may need a more descriptive name. I'm being slightly hard-assed about this because this kind of thing has caused all kinds of trouble in OCS (human-readable vs. TCS-grokkable names, with no explanation anywhere).